src/App.tsx

```diff
@@ -14,191 +67,23 @@ export const App: React.FC = () => {
       <h1 className="todoapp__title">todos</h1>
 
       <div className="todoapp__content">
-        <header className="todoapp__header">
-          {/* this button should have `active` class only if all todos are completed */}
-          <button
-            type="button"
-            className="todoapp__toggle-all active"
-            data-cy="ToggleAllButton"
-          />
-
-          {/* Add a todo on form submit */}
-          <form>
-            <input
-              data-cy="NewTodoField"
-              type="text"
-              className="todoapp__new-todo"
-              placeholder="What needs to be done?"
+        <Header />
+        {!!todos.length && (
+          <>
+            <TodoList todos={filteredTodos} />
+            <Footer
+              value={filterValue}
+              itemsLeft={todos.filter(todo => !todo.completed).length}
+              onFilter={onFilterHandler}
```

Comment:
```suggestion
              onFilter={setFilterValue}
```
`onFilterHandler` function wrapper is redundant